db/seeds.rb

```diff
@@ -5,3 +5,28 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Role.find_or_create_by(name: 'observer')
+Role.find_or_create_by(name: 'client_admin')
+Role.find_or_create_by(name: 'admin')
+
+User.with_email_role_slug!(
+  ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
```

Comment:
When this was a migration, it didn't matter, because it only ran once, when the env vars were correctly defined.

Now that it runs on every deploy via seeds, yes you are correct, the env vars must continue to be correctly set in production or we will get test users for production budget approvers. That sort of defeats the purpose of moving away from env vars, if they must continue to be set indefinitely just for deploys to work.